pyspark_datasources/__init__.py

```
@@ -5,6 +5,7 @@
 from .huggingface import HuggingFaceDatasets
 from .kaggle import KaggleDataSource
 from .opensky import OpenSkyDataSource
+from .robinhood import RobinhoodDataSource
```

Ruff (0.12.2)

`.robinhood.RobinhoodDataSource` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)